util/envDynawo.sh

```diff
@@ -644,7 +646,7 @@ config_3rd_party() {
   if [ $DYNAWO_LIBARCHIVE_HOME_DEFAULT != true ]; then
     CMAKE_OPTIONAL="$CMAKE_OPTIONAL -DLIBARCHIVE_HOME=$DYNAWO_LIBARCHIVE_HOME"
   fi
-  case $DYNAWO_BUILD_TYPE_THIRD_PARTY in
+  case $DYNAWO_BUILD_TYPE in
```

Comment:
Why not testing DYNAWO_BUILD_TESTS?

Reply:
yes test should be similar as the one in config_dynawo

Reply:
OK